### PR TITLE
Add Jiralert app to Security Pack

### PR DESCRIPTION
**Pull request by fhielpos**
This PR:

- adds `Jiralert` app to the `security-pack`
- updates `starboard-exporter` to 0.4.1

### Testing

Description on how security-pack can be tested.

- [ ] fresh install works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM
- [ ] upgrade from previous version works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Comment by fhielpos** [comment]
@stone-z , I also bumped starboard-exporter if thats okay

We will need to promote `jiralert` to the Giantswarm catalog.

Also not sure if `chartName` should be `jiralert-app` or just `jiralert`

**Comment by fhielpos** [comment]
Updated the schema

File: helm/security-pack/values.yaml [hunk 1]
```diff
@@ -26,6 +26,14 @@ apps:
     namespace: security-pack
     version: 0.3.2
 
+  jiralert:
+    appName: jiralert
+    chartName: jiralert
+    catalog: giantswarm
```
Comment (stone-z) [inline helm/security-pack/values.yaml]: Need to release jiralert app in giantswarm catalog before merging with this

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `jiralert-app` 0.0.1 to the security pack for creating Jira tickets.
```
Comment (stone-z) [inline CHANGELOG.md:12]:
```suggestion
- Add `jiralert-app` 0.0.2 to the security pack for creating Jira tickets.
```

**Review by stone-z — APPROVED** [review]
LGTM but please update the version in the changelog

**Pull request merged**